services/ui-src/src/components/fields/DynamicField.tsx

```diff
@@ -369,8 +381,9 @@ const sx = {
     marginLeft: "0.625rem",
   },
   removeImage: {
-    width: "1.25rem",
-    height: "1.25rem",
+    maxWidth: "none",
```

Comment:
when I bumped up the height and width to `1.5rem` the width was getting squished to `22.12px` or something. Chakra is setting `max-width: 100%` on the Image component and somehow disabling that fixes the issue so they both fill out the 24px. Anyone have other ideas to address this?